_episodes/05-loops-and-functions.md

```diff
@@ -538,11 +538,11 @@ Which would return:
 Change the value of `a` to see how this function works. The statement `elif`
 means "else if", and all of the conditional statements must end in a colon.
 
-The if loops in the function `yearly_data_arg_test` check whether there is an
+The if statements in the function `yearly_data_arg_test` check whether there is an
 object associated with the variable names `start_year` and `end_year`. If those
-variables are `None`, the if loops return the boolean `True` and execute whaever
+variables are `None`, the if statements return the boolean `True` and execute whatever
```

Comment:
Not a problem introduced in this P.R., but looking at this I'm noticing some problems with the wording. I think what is meant here is that if the variables are `None`, the if *condition* (i.e. the thing between `if` and the colon) *evaluates to* the boolean `True`.

It's a small difference, but I think it's important to reserve the verb "return" for talking about the results of functions/methods.